editor/plugins/animation_player_editor_plugin.cpp

```diff
@@ -896,6 +897,12 @@ void AnimationPlayerEditor::_update_player() {
 	_update_animation();
 }
 
+void AnimationPlayerEditor::_update_frame_step(float p_len) {
+	if (!p_len)
```

Comment:
Please be explicit instead of relying on a cast to bool. `if (p_len == 0)`.
What happens if `p_len` is negative? BTW, this should be `p_step`.